integrations/localhero/src/main/java/ch/stayhome/integrations/localhero/job/LocalHeroPostProcessor.java

```diff
@@ -9,30 +16,48 @@
 
 @Slf4j
 public class LocalHeroPostProcessor implements ItemProcessor<LocalHeroPost, StayHomeEntry> {
-    private final LocalHeroProperties config;
-
-    LocalHeroPostProcessor(final LocalHeroProperties config) {
-        this.config = config;
-    }
-
-    @Override
-    public StayHomeEntry process(LocalHeroPost item) {
-        return StayHomeEntry.builder()
-                .id(String.valueOf(item.getId()))
-                .name(item.getTitle().getRendered())
-                .description(item.getExcerpt().getRendered())
-                .website(item.getGuid().getRendered())
-                .providerName(config.getProviderName())
-
-                .location("Bern")                   // FIXME: don't hard code, there are multiple locations
-                .categories("local-hero.ch")        // FIXME: don't know yet what to put in here
-                .delivery("Bern")                   // FIXME: What's the exact requirement for this
-
-                .contact("")
-                .email("")
-                .phone("")
-                .ttl(config.getTtlSeconds())
-
-                .build();
-    }
+
+	private final SourceConfig sourceConfig;
+
+	private final LocalHeroChApi localHeroChApi;
+
+	private final Long defaultTTL;
+
+	private final Map<String, LocalHeroCategory> categoryCache = new HashMap<>();
+
+	LocalHeroPostProcessor(SourceConfig sourceConfig, LocalHeroChApi localHeroChApi, Duration defaultTTL) {
+		this.sourceConfig = sourceConfig;
+		this.localHeroChApi = localHeroChApi;
+		this.defaultTTL = defaultTTL.getSeconds();
+	}
+
+	@Override
+	public StayHomeEntry process(LocalHeroPost item) {
+		return StayHomeEntry.builder()
+				.id(String.valueOf(item.getId()))
+				.name(item.getTitle().getRendered())
+				.description(item.getExcerpt().getRendered())
+				.website(item.getGuid().getRendered())
+				.providerName(this.sourceConfig.getProviderName())
+				.location(this.sourceConfig.getPlace())
+				.categories(determineCategories(item))
+				.delivery(this.sourceConfig.getPlace())
+				.contact("")
+				.email("")
+				.phone("")
+				.ttl(defaultTTL)
+				.build();
+	}
+
+	private String determineCategories(LocalHeroPost item) {
```

Comment:
@lugipfupf This is where I tried to resolve the category ids from localhero to get the name and then join everything into the categories field that the sync-service needs